libs/domains/observability/feature/src/lib/service-overview/card-metric/card-metric.tsx

```diff
@@ -0,0 +1,103 @@
+import { type IconName } from '@fortawesome/fontawesome-common-types'
+import clsx from 'clsx'
+import { type ComponentProps, type ReactNode } from 'react'
+import { Button, Icon, Skeleton, Tooltip } from '@qovery/shared/ui'
+import { twMerge } from '@qovery/shared/util-js'
+
+interface CardMetricProps extends Omit<ComponentProps<'button'>, 'value'> {
+  title: string
+  value: number | ReactNode
+  unit?: string
+  status: 'GREEN' | 'YELLOW' | 'RED'
+  description?: string
+  isLoading?: boolean
+  hasModalLink?: boolean
+  icon?: IconName
+}
+
+export function CardMetric({
+  title,
+  value,
+  unit,
+  status,
+  description,
+  isLoading = true,
+  className,
+  hasModalLink = false,
+  icon = 'chart-line',
+  onClick,
+  ...props
+}: CardMetricProps) {
+  const getStatusColor = () => {
```

Comment:
As the CardMetrics component will rerender quite often, using `useMemo` for these 2 getters could improve the performance a bit